linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_exec_shield/bash/shared.sh

```diff
@@ -15,11 +15,11 @@ if [ $(getconf LONG_BIT) = "32" ] ; then
   replace_or_append '/etc/sysctl.conf' '^kernel.exec-shield' '1' '@CCENUM@'
 fi
 
-if [ $(getconf LONG_BIT) = "64" ] ; then
+if [ "$(getconf LONG_BIT)" = "64" ] ; then
   if grep --silent noexec /boot/grub2/grub*.cfg ; then 
         sed -i "s/noexec.*//g" /etc/default/grub
         sed -i "s/noexec.*//g" /etc/grub.d/*
         GRUBCFG=`ls | grep '.cfg$'`
```

Comment:
This is really strange, as the PR doesn't pay attention to location when `ls` is executed, I wonder what is supposed to end up in the `GRUBCFG` file.